README.md

```diff
@@ -8,24 +8,24 @@ for those systems, and specified telemetry or data from various tools (`sar`,
 `vmstat`, `perf`, etc.).
 
 The second sub-system is the `pbench-server`, which is responsible for
-archiving result tar balls, indexing them, and unpacking them for display.
+archiving result tar balls, indexing them, and managing access to their
+contents.
+It provides a RESTful API which can be used by client applications, such as
+the Pbench Dashboard, which provides a web-based interface to the Pbench
+Server.
```

Comment:
Trying again: the RESTful API stuff is good, having the dashboard reference in the middle of the sentence is distracting and isn't needed here. Ideally, "Pbench Dashboard" is a distinct subsystem deserving of its own paragraph, and if you want to mention that it's a RESTful API client, that's fine.

(And really, though it's outside of where GitHub lets me comment, this "web-server" subsystem is a total crock. The server packages a few `npm` modules and the `jschart.js` script to support displays, but there's nothing here in any way resembling a "web server" ... we're just stocking some browser snacks, and it's all part of the Pbench Server package. With Quisby, maybe nobody will even care anymore. 🤞🏻 )

Reply:
Still don't like the dashboard phrase and I think it complicates without helping any; but the overall paragraph is improved somewhat. I give up.